internal/oci/runtime_oci.go

```diff
@@ -625,6 +628,7 @@ func (r *runtimeOCI) StopContainer(ctx context.Context, c *Container, timeout in
 		}
 		err = waitContainerStop(ctx, c, time.Duration(timeout)*time.Second, true)
 		if err == nil {
+			c.SetFinished(time.Now())
```

Comment:
This shouldn't be added. We already set finished time from the exit file time stamp.